### PR TITLE
feat: support ipv6 for kubernetes

### PR DESCRIPTION
**Pull request by ajasnosz**
# Description

Adds to SC4SNMP support for polling and receiving traps notifications from IPv6 devices.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature

## How Has This Been Tested?

unit, integration and manual tests

## Checklist

- [x] My commit message is [conventional](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] I have run pre-commit on all files before creating the PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: splunk_connect_for_snmp/traps.py [hunk 1]
```diff
@@ -49,7 +49,7 @@
 SECURITY_ENGINE_ID_LIST = os.getenv("SNMP_V3_SECURITY_ENGINE_ID", "80003a8c04").split(
     ","
 )
-
+IPv6_ENABLED = True if "ipv6" in os.getenv("IPv6_ENABLED", "[IPv4]").lower() else False
```
Comment (ikheifets-splunk) [inline splunk_connect_for_snmp/traps.py:52]: probably we can make it bool, anyway we assigning it on k8s based on ipFamily:

```
            - name: IPv6_ENABLED
               value: {{ .Values.traps.ipFamilies | quote }}
```

File: rendered/manifests/tests/splunk-connect-for-snmp/templates/traps/service.yaml [hunk 2]
```diff
@@ -12,11 +12,14 @@ metadata:
     app.kubernetes.io/managed-by: Helm
   annotations:
     metallb.universe.tf/allow-shared-ip: "splunk-connect"
+    metallb.universe.tf/loadBalancerIPs: 10.202.6.213
```
Comment (ikheifets-splunk) [inline rendered/manifests/tests/splunk-connect-for-snmp/templates/traps/service.yaml:15]: I don't how it's working but it's strange that for test we using hardcoded IP of LB

**Review by ikheifets-splunk — COMMENTED** [review]
am I understood correctly that's only k8s? Or for docker it's working out of the box

File: rendered/manifests/tests/splunk-connect-for-snmp/templates/traps/service.yaml (hunk 2 above)
Comment (ajasnosz) [reply rendered/manifests/tests/splunk-connect-for-snmp/templates/traps/service.yaml:15]: this is just rendered examples. For the tests the ip is taken when building the environment.

**Comment by ajasnosz** [comment]
> am I understood correctly that's only k8s? Or for docker it's working out of the box

I've changed the pr name to match what it is. That part is only for k8s, for docker it will be done in different pr

File: splunk_connect_for_snmp/traps.py (hunk 1 above)
Comment (ajasnosz) [reply splunk_connect_for_snmp/traps.py]: changed to have it in chart

**Review by ikheifets-splunk — APPROVED** [review]

**Pull request merged**